File: app/src/main/java/name/monwf/customiuizer/tasker/UnlockReceiver.java
package name.monwf.customiuizer.tasker;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.os.Bundle;

import name.monwf.customiuizer.mods.GlobalActions;

public class UnlockReceiver extends BroadcastReceiver {

	@Override
	public void onReceive(Context context, Intent intent) {
		Bundle bundle = intent.getBundleExtra(Constants.EXTRA_BUNDLE);
		if (bundle != null) {
			Intent sendIntent = new Intent();
			sendIntent.setAction(GlobalActions.ACTION_PREFIX + "UnlockSetForced");
			sendIntent.putExtras(bundle);
			context.sendBroadcast(sendIntent);
		}
	}

}

File: app/src/main/java/tv/withaibuild/customiuizer/tasker/UnlockReceiver.kt
package tv.withaibuild.customiuizer.tasker

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import tv.withaibuild.customiuizer.mods.GlobalActions

class UnlockReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        val bundle = intent.getBundleExtra(Constants.EXTRA_BUNDLE)
        if (bundle != null) {
            val sendIntent = Intent().apply {
                action = GlobalActions.ACTION_PREFIX + "UnlockSetForced"
                putExtras(bundle)
            }
            context.sendBroadcast(sendIntent)
        }
    }
}
